kinisi/parser.py

```diff
@@ -52,7 +52,7 @@ class Parser:
     def __init__(self,
                  disp: np.ndarray,
                  indices: np.ndarray,
-                 framework_indices: np.ndarray,
+                 drift_indices: np.ndarray,
```

Comment:
Can you remind me of the logic for renaming this again?

Reply:
Nevermind, I have scrolled down and remember.

Reply:
This is renamed so that framework_indices can be used as a keyword argument elsewhere.